package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@microsoft/mezzurite-core",
+  "version": "2.0.0-alpha",
+  "description": "Core library for Mezzurite data and event communication",
+  "main": "./lib/mezzurite.js",
+  "unpkg": "./dist/mezzurite.js",
```

Comment:
Could you remind me what the difference was between `"main"`, `"unpkg"`, and `"module"` here? I assume it's something around UMD vs ES6?

Reply:
Yup, it's for indicating which versions of the built files are used in a UMD vs ES6 vs MJS context.